core/src/main/scala/epollcat/internal/ch/SocketHelpers.scala

```diff
@@ -89,4 +89,9 @@ private[ch] object SocketHelpers {
     new InetSocketAddress(inetAddr, port)
   }
 
+  // Return text translation of getaddrinfo (gai) error code.
+  def getGaiErrorMessage(gaiErrorCode: CInt): String = {
+    s"${fromCString(posix.netdb.gai_strerror(gaiErrorCode))}"
```

Comment:
I think we can simplify this :)

```suggestion
    fromCString(posix.netdb.gai_strerror(gaiErrorCode))
```

Reply:
Done, thank you.